resources/sql/queries.sql

```diff
@@ -296,9 +296,9 @@ WHERE id = :id;
 
 -- :name create-license-attachment! :insert
 INSERT INTO license_attachment
-(modifierUserId, filename, type, data)
+(modifierUserId, filename, type, data, start)
```

Comment:
are you planning to add the same code to ```organisations``` ?

Reply:
It's there already! It turns out the default values in audit_log and organization were not used: REMS always specified the value explicitly.

Reply:
ok, so we did have this approach already? Good to refactor then!

Reply:
Yeah it was kind of a belt-and-suspenders thing where most timestamps were created in REMS, but some were left up to the db defaults.

Reply:
The db defaults make sense for helping with the SQL console use. I think they can be left out. The middle ground is to have the defaults and always make sure you don't use the default in the app.